app/controllers/use_case_controller.rb

```diff
@@ -66,6 +60,18 @@ def attribute_changes?
     !@model_instance.changed_attributes.blank?
   end
 
+  def build_model_instance
+    if params[:action] == 'new'
```

Comment:
wha about putting this conditional into a method named `filtered_params` and then leaving L69 as-is?